docs/src/main/asciidoc/_admin-guide/CLI_Recipes.adoc

```diff
@@ -190,14 +190,19 @@ Assume you have a host that is called "primary"
 
 [source,options="nowrap"]
 ----
-[domain@localhost:9990 /] /host=primary:read-config-as-xml
```

Comment:
The new text should be added and the old one should NOT be removed now, since the `read-config-as-xml` still exists.